code/common/datadog.q

```diff
@@ -0,0 +1,22 @@
+// DATADOG CHECKS
+
+//Create datadog namespace
+\d .dg
+
+//Functions are set to return 1b or 0b based on the health of the service.
+//Default check returns 1b from each process to indicate process is up and can be queried.
+//These checks run every time crontab/timer is run.
+
+handlers:(`$())!()
+
+is_ok:{
+  f:$[.proc.proctype in key .dg.handlers;.dg.handlers .proc.proctype;1b]
```

Comment:
Can skip creating variable `f` + make one line + this naming convention is inconsistent, use `isok` or `isOK` (as appropriate): 
```suggestion
isok:{$[.proc.proctype in key .dg.handlers;.dg.handlers .proc.proctype;1b]}
```